scripts/spawn_sdf_test.py

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/python
+from ed_object_models import spawn_sdf_in_gazebo
```

Comment:
If it is a test put it in the test folder.

Reply:
It is more of a simple example of how to use it. Maybe put it into (currently not existing) readme file for this package? Or would a test folder still be the better option?

Reply:
If this keeps a test, put it in the test folder. Also would be nice to have this feature fully implemented for robotics_testlabs incl. all environment variables, launch files etc.